main/css/base.css

```diff
@@ -5315,3 +5315,109 @@ i.size-32.icon-new-work{
 #settings .span6:nth-child(2n+1) {
     clear:left
 }
+
+/* CSS for the view by session My Course tab */
+.sessionViewBlock {
+    font-size : 14px;
+    font-weight : normal;
+}
+
+.sessionViewRow {
+    padding: 8px 0;
+}
+
+.sessionViewLvl2 {
+    margin-left : 66px;
+}
+
+.sessionViewLvl3 {
+    padding-left : 92px;
+}
+
+.sessionViewLvl4 {
+    padding-left : 102px;
+}
+
+.sessionViewLvl5 {
+    padding-left : 128px;
+    font-weight : normal;
+}
+
+.sessionViewLvl6 {
+    padding-left : 20px;
+}
+
+.sessionViewLvl7 {
+    padding-left : 48px;
+}
+
+img.sessionView {
+    padding-right: 10px;
+}
+
+a.sessionView {
+    font-weight : bold;
+}
+
+.sessionViewUserCat {
+    font-weight : bold;
+    padding-bottom : 10px;
+}
+
+.sessionViewSessionCat {
+    font-weight : bold;
+    padding-bottom : 10px;
+}
+
+.sessionViewSession {
+    font-weight : bold;
+}
+
+/* Like the well class */
+.sessionViewWell {
+    min-height: 20px;
+    padding: 19px;
+    margin-bottom: 20px;
+    background-color: #F5F5F5;
+    border: 1px solid rgba(0, 0, 0, 0.05);
+    border-radius: 4px;
+    box-shadow: 0px 1px 1px rgba(0, 0, 0, 0.05) inset;
+}
+
+.sessionViewButton {
+    border : none;
+    min-width : 28px;
+}
+
+.viewbySessionLink {
+    text-align: right;
+}
+
+p.viewbySessionLink a {
+    padding-right : 10px;
+    padding-left : 10px;
+}
+
+.viewBySessionActive {
+    text-decoration : underline;
+}
+
+.viewbySessionTeachers {
+    display : none; /* change this if you want to display teachers name in this view */
+    font-size : 10px;
+    overflow-x : auto;
+    overflow-y: auto;
+    max-height:50px;
+}
+
+.viewbySessionRightActions {
+    float : right;
+}
+
+.viewbySessionRightActions img {
+    width : 12px;
+}
+
+.sessionViewSessionGoToCourseInSession {
```

Comment:
By virtue of a recent update to the coding conventions, all CSS styles should be lower-case-with-dash. Sorry :-)